cli/standalone/mod.rs

```diff
@@ -664,7 +657,7 @@ pub async fn run(
   };
 
   let permissions = {
-    let mut permissions = metadata.permissions.to_options();
+    let mut permissions = metadata.permissions.to_options(&[]);
```

Comment:
I assume this means that standalone binaries won't be able to import anything from any hosts - which is fine because they currently don't support any dynamic imports. Maybe leave a comment for future reference?

Reply:
It means that we don't infer any cli arguments. I'll add a comment.